lib/infoblox/resource/host.rb

```diff
@@ -10,6 +10,8 @@ class Host < Resource
                          :name,
                          :view
 
+    remote_attr_accessor :ipv6addrs if ENV['WAPI_VERSION'].to_f >= 1.1
```

Comment:
Instead of ENV['WAP_VERSION'], I'd prefer to use the module function `Infoblox.wapi_version` - to consolidate the places that this configuration comes from.  You can see how that is used in `lib/infoblox/resource.rb` in the `self._return_fields` method. 

Also, you don't need to do this float business - string comparison will work fine (`'1.2' > '1.1'` is true).